docs/pages/reference/helm-reference/teleport-plugin-mattermost.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: teleport-plugin-mattermost Chart Reference
+description: Values that can be set using the teleport-plugin-mattermost Helm chart
+---
+
+The `teleport-plugin-mattermost` Helm chart is used to configure the MatterMost Teleport plugin which allows users to receive access requests to channels or as direct messages via MatterMost.
+
+You can [browse the source on GitHub](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/mattermost).
+
+This reference details available values for the `teleport-plugin-mattermost` chart.
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## `teleport.address`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+This parameter contains the host/port combination of the teleport auth server.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretName`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Name of the Kubernetes secret that contains the credentials for the connection.
+
+The secret should be in the following format:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: teleport-plugin-mattermost-identity
+data:
+  auth_id: ...
+```
+
+Check out the [MatterMost Helm Chart documentation](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/mattermost#prerequisites) for more information about how to acquire these credentials!
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    identitySecretName: "teleport-plugin-mattermost-identity"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.identitySecretName="teleport-plugin-mattermost-identity"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretPath`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `"auth_id"` | No |
+
+Name of the key in the Kubernetes secret that holds the credentials for the connection. If the secret follows the format above, it can be omitted.
```

Comment:
```suggestion
Name of the key in the Kubernetes secret that holds the credentials for the connection to your Teleport cluster. If the secret has the path, `"auth_id"`, you can omit this field.
```

Reply:
Also, I think the difference between this and `identitySecretName` can be clearer. One is the name of a secret and one is a key in the secret's `data` field?

Reply:
I rephrased it a bit, I think it's clearer now